packages/fluentui/react-northstar/src/themes/teams/components/Tree/treeTitleStyles.ts

```diff
@@ -12,6 +14,52 @@ const treeTitleStyles: ComponentSlotStylesPrepared<TreeTitleStylesProps, TreeTit
     position: 'relative',
     ...getBorderFocusStyles({ variables: siteVariables }),
   }),
+
+  selectionIndicator: ({ props: p, variables: v }): ICSSInJSStyle => ({
+    display: 'inline-block',
+    float: 'right',
+    boxShadow: 'unset',
+    width: pxToRem(16),
+    height: pxToRem(16),
+
+    borderColor: v.borderColor,
+    borderStyle: v.borderStyle,
+    borderRadius: v.borderRadius,
+    borderWidth: v.borderWidth,
+    color: v.indicatorColor,
+    margin: v.selectionIndicatorMargin,
+    padding: v.padding,
+    userSelect: 'none',
+
+    backgroundImage: checkboxIndicatorUrl(v.indicatorColor, v.background),
+    backgroundPosition: 'center',
+    backgroundRepeat: 'no-repeat',
+
+    ...(p.selected && {
+      borderColor: v.checkedBorderColor,
+      backgroundImage: checkboxIndicatorUrl(v.checkedIndicatorColor, v.checkedBackground),
+    }),
+
+    ...(p.disabled && {
+      background: v.disabledBackground,
+      borderColor: v.disabledBorderColor,
+    }),
+
+    ...(p.disabled &&
+      p.selected && {
```

Comment:
can't this be added  in the `p.disabled` above ?

Reply:
I took this logic from the checkboxStyles.ts 
There is on line 115 p.checked what in tree meaning is selected
So I believe it has some sense, even all the "disable" state I think will be next big chapter :) because I am not sure now how it should behave? only checkbox should be disabled? or whole tree item? these questions need be answer in furture. :)